packages/@aws-cdk/aws-lambda/lib/languages/node.ts

```diff
@@ -0,0 +1,52 @@
+import cdk = require('@aws-cdk/cdk');
+import { Function as LambdaFunction, PartialFunctionProps } from '../function';
+import { Runtime } from '../runtime';
+import { LambdaBuilderCode } from './lambda-builder';
+
+export class NodeVersion {
+  public static readonly NodeJS = new NodeVersion(Runtime.NodeJS);
+  public static readonly NodeJS43 = new NodeVersion(Runtime.NodeJS43);
```

Comment:
It is probably not worth adding support for Node4.3 here as it is depreciated on Lambda: https://docs.aws.amazon.com/lambda/latest/dg/runtime-support-policy.html